src/components/balances/stx-balance-card.tsx

```diff
@@ -18,21 +18,36 @@ import { useAtomValue } from 'jotai/utils';
 import { accountInViewTokenOfferingData } from '@store/currently-in-view';
 import { MODALS } from '@common/constants';
 import { useAppDispatch } from '@common/state/hooks';
+import { useCurrentStxPrice } from '@common/utils/market-utils';
+
+const usdFormatter = new Intl.NumberFormat('en-US', {
+  style: 'currency',
+  currency: 'USD',
+});
 
 export const BalanceItem = ({ balance, ...rest }: any) => {
+  const { data: stxPrice } = useCurrentStxPrice();
+
+  const formattedBalance = balance.toLocaleString(undefined, { minimumFractionDigits: 2, maximumFractionDigits: 6 });
```

Comment:
Changed this component to use the raw number for the `balance` prop rather than the formatted string for the most accuracy when calculating USD value